dm/proto/tracer_syncer.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package pb;
+
+import "tracer_type.proto";
+
+
+message BaseEvent {
```

Comment:
put this `BaseEvent` into another proto file rather than `tracer_syncer.proto`? maybe this is not a problem.

Reply:
add a `tracer_base.proto` file